src/ml/neural_net/tf_compute_context.cpp

```diff
@@ -317,14 +318,22 @@ std::unique_ptr<model_backend> tf_compute_context::create_activity_classifier(
   const float* pred_window = prediction_window.data();
   int pw = static_cast<int>(*pred_window);
 
+  int seed = FLEX_UNDEFINED;
```

Comment:
`FLEX_UNDEFINED` doesn't have `int` type. Can we just require that the `config` continue the random seed?

Reply:
What do you mean? Inference config has no random seed. I just have to pass something to python side, it won't be used at inference anyway.